customer/views.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import viewsets, mixins
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.permissions import IsAuthenticated
+
+from core.models import Invoice
+
+from customer import serializers
+
+
+class InvoiceViewSet(viewsets.GenericViewSet, mixins.ListModelMixin):
```

Comment:
You can just inherit from viewsets.ModelViewSet since all the actions are required in this case not just listing. You may refer to the [source code](https://github.com/encode/django-rest-framework/blob/master/rest_framework/viewsets.py#L235) to understand how `ModelViewSet` works or you may refer to [this diagram](https://drive.google.com/file/d/19tGSTJ43xoTj7jM-0PxjQWwCJxSqgui6/view?usp=sharing) to learn more about the inheritance hierarchy of viewsets and views.